docs/auth0_universal-login_prompts_update.md

```diff
@@ -5,10 +5,18 @@ layout: default
 
 Update the custom text for a prompt.
 
+## Usage
 ```
 auth0 universal-login prompts update [flags]
 ```
 
+## Examples
+
+```
+  auth0 universal-login prompts update <prompt> --language es
```

Comment:
Should the first example (`auth0 universal-login prompts update <prompt> --language es`) be `auth0 universal-login prompts update <prompt> --language <language>`?

Reply:
Should we also add an example with the `ul` alias? Like in the examples for `auth0 universal-login show`.